### DISCUSSION TITLE
if cleanup don't do anything,how can I stop send notification?

### DISCUSSION THREAD

**Discussion by lengjunbin-trec** [spatie/laravel-backup#1457] [Q&A] [answered]
I execute `php artisan backup:clean` to clean old backup zip file,if there is not old backup file,I don't want to send notification,only while delete something ,send nontofication

**Comment by lengjunbin-trec** [comment, marked as answer]
@erikn69

**Reply by erikn69** [reply]
Fell free to make a PR to fix that
